api/magic_links/routes.py

```diff
@@ -177,16 +179,25 @@ def use(self, link_id: str):
                     account_id=link.get("accountId"),
                     redirect_url=link.get("redirectUrl"),
                 )
-            return error_response(403, "Link expired")
-        return error_response(403, "Link expired or invalid")
+            return redirect(
+                env.config.get("INVALID_MAGIC_LINK_URL") + "?e=Link+expired"
```

Comment:
Can't we use `url_for` to derive this route rather than getting it from config? (Sorry if I missed something)

Reply:
I concur